### PR TITLE
Improvement of web proxy client code

### PR DESCRIPTION
**Pull request by Shokman**
This includes a merge of my previosu improvements with Pablo video work.
This includes:
- pep8 compilant code
- better ROS adapted code
- modified launch file used by concert_master

### CODE REVIEW

File: rosbridge_server/launch/rosbridge_websocket.launch [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <arg name="authenticate" default="false" />
   
   <group if="$(arg ssl)">
-    <node name="rosbridge_websocket" pkg="rosbridge_server" type="rosbridge_websocket" output="screen">
+    <node name="rosbridge_websocket_client" pkg="rosbridge_server" type="rosbridge_websocket_client" output="screen">
```
Comment (adamantivm) [inline rosbridge_server/launch/rosbridge_websocket.launch]: @Shokman Is this on purpose? This is going to break the regular websocket server!

**Comment by Shokman** [comment]
@adamantivm please review changes

File: rosbridge_server/launch/rosbridge_websocket_client.launch [hunk 2]
```diff
@@ -4,6 +4,7 @@
   <arg name="certfile" />
   <arg name="keyfile" />
   <arg name="authenticate" default="false" />
+  <arg name="webserver_address" default="ws://localhost:9090" />
```
Comment (adamantivm) [inline rosbridge_server/launch/rosbridge_websocket_client.launch]: @Shokman  sorry to be annoying, can we please change this variable name to webserver_url or webserver_uri? I insist that the value is not an address but an URL (i.e.: it includes a protocol, a port and a path).
Also, please make the default ws://localhost:9090/ws to match with https://github.com/creativa77/rocon_web_proxy_server/blob/master/main.py#L97

**Comment by Shokman** [comment]
@adamantivm refactoring done

**Comment by adamantivm** [comment]
Thank you @Shokman :+1:

**Pull request merged**